docs/content/preview/yugabyte-voyager/migrate/migrate-steps.md

```diff
@@ -18,18 +18,19 @@ The following page describes the steps to perform and verify a successful offlin
 
 ## Migration workflow
 
-![Offline migration workflow](/images/migrate/migration-workflow-new.png)
+![Offline migration workflow](/images/migrate/migration-workflow-updated.png)
```

Comment:
@sudhanshu456 we will need the `assess-migration` step to be introduced in every other workflow (live, live-fall-back, live-fall-forward) as well.

Reply:
do you mean in the images as well?

Reply:
yes, in the images as well